web/client/resources/js/goconvey.js

```diff
@@ -769,7 +798,9 @@ function process(data, status, jqxhr)
 	convey.intervalFuncs.momentjs();
 
 	// Show notification, if enabled
-	if (notif())
+	var levelRegex = new RegExp("("+convey.notificationLevel+")", "i");
+	var levelEnabled = current().overall.status.class.match(levelRegex);
```

Comment:
isn't this a bit overkill? This implies that data-notification-level is actually a regex fragment.

If this is a good approach, then why have 'all' and not just '.*' ?